examples/framesjs-starter/app/examples/new-api-custom-middleware/frames/frames.ts

```diff
@@ -0,0 +1,13 @@
+import { createFrames } from "frames.js/next";
+
+export const frames = createFrames({
+  basePath: "/examples/new-api-custom-middleware/frames",
+  middleware: [
+    async (ctx, next) => {
+      const body = await ctx.request.clone().json();
+
+      return next({ body });
+    },
+    // the request body will now be available via ctx.body in your handler
+  ],
```

Comment:
can we make this middleware add something that is used in the next frame?

maybe even just a hardcoded string

would demonstrate the use case more succinctly